bedrock/mozorg/forms.py

```diff
@@ -506,6 +510,238 @@ def __init__(self, *args, **kwargs):
+        required=False,
+        widget=forms.TextInput(
+            attrs={
+                'size': 20
+            }
+        )
+    )
+    street = forms.CharField(
+        required=False,
+        widget=forms.Textarea(
+            attrs={
+                'rows': '',
+                'cols': ''
+            }
+        )
+    )
+    city = forms.CharField(
+        required=False,
+        max_length=40
+    )
+    state = USStateField(
+        required=False,
+        initial='',
+        widget=USStateSelectBlank()
+    )
+    province = forms.CharField(
+        required=False,
+        max_length=40
+    )
+    country = forms.ChoiceField(
+        required=True,
+    )
+    zip = forms.CharField(
+        required=False,
+        max_length=40
+    )
+    campaign_type_description = forms.CharField(
+        required=False,
+        widget=forms.Textarea(
+            attrs={
+                'rows': '',
+                'cols': ''
+            }
+        )
+    )
+    interested_countries = forms.CharField(
+        required=False,
+        widget=forms.Textarea(
+            attrs={
+                'rows': '',
+                'cols': ''
+            }
+        )
+    )
+    interested_languages = forms.CharField(
+        required=False,
+        widget=forms.Textarea(
+            attrs={
+                'rows': '',
+                'cols': ''
+            }
+        )
+    )
+    industry = forms.ChoiceField(
+        choices=industries,
+        required=False,
+        widget=forms.Select(
+            attrs={
+                'title': _lazy('Industry'),
+                'size': 1
+            }
+        )
+    )
+    campaign_type = forms.ChoiceField(
+        choices=(
+            ('', _lazy(u'Select Campaign Type')),
+            ('Brand', _lazy(u'Brand')),
+            ('Direct Response', _lazy(u'Direct Response')),
+            ('Other', _lazy(u'Other'))
+        ),
+        required=False,
+        widget=forms.Select(
+            attrs={
+                'title': _lazy('Campaign Type')
+            }
+        )
+    )
+    # honeypot
+    office_fax = forms.CharField(widget=HoneyPotWidget, required=False)
+    # uncomment below to debug salesforce
+    # debug = forms.IntegerField(required=False)
+    # debugEmail = forms.EmailField(required=False)
+
+    def __init__(self, *args, **kwargs):
+        kwargs.pop('lead_source', None)
+        super(ContentServicesForm, self).__init__(*args, **kwargs)
+        locale = kwargs.get('locale', 'en-US')
+        country_list = product_details.get_regions(locale).items()
+        country_list = sorted(country_list, key=lambda country: country[1])
+        country_list.insert(0, ('', ''))
+        self.fields['country'].choices = country_list
+
+    def clean(self):
+        data = super(ContentServicesForm, self).clean()
+        if data.get('country') == 'us' and not data.get('state'):
+            raise ValidationError(self.fields['state'].error_messages['invalid'])
```

Comment:
If JS is disabled (I know, I know) and the user is in the US, there is no way to avoid this error.